ee/ucp/deploy-application-package.md

```diff
@@ -8,46 +8,46 @@ keywords: ucp, swarm, kubernetes, application, app package
 
 Docker Enterprise Edition 2.1 introduces application packages in Docker. With application packages, you can add metadata and settings to an existing Compose file. This gives operators more context about applications they deploy and manage.
 
-An application package can have one of these formats:
+Application packages can present in one of two different formats, Directory or Single-file:
 
-- **Directory format**: Defined by metadata.yml, a docker-compose.yml, and a settings.yml files inside a `my-app.dockerapp` folder. This is also called the folder format.
-- **Single-file format**: Defined by metadata.yml, docker-compose.yml, and settings.yml concatenated in that order and separated by `---\n` in a single file named named `my-app.dockerapp`.
+- **Directory**: Defined by metadata.yml, a docker-compose.yml, and a settings.yml files inside a `my-app.dockerapp` folder. This is also called the folder format.
+- **Single-file**: Defined by metadata.yml, docker-compose.yml, and settings.yml concatenated in that order and separated by `---\n` in a single file named named `my-app.dockerapp`.
 
 Once an application package has been deployed, you manipulate and manage it as you would any stack.
 
 ## Creating a stack in the UCP web interface
 
-To create a stack in the UCP web interface, follow these steps:
+To create a stack in the UCP web interface:
 
-1. Go to the UCP web interface.
-2. In the lefthand menu, first select **Shared Resources**, then **Stacks**.
+1. Access the UCP web interface.
+2. From the left menu, select **Shared Resources** > **Stacks**.
 
-    ![Create stacks in UCP](/ee/ucp/images/ucp-create-stack.png)
+    ![Create stacks in UCP](/ee/ucp/images/v32stacks.png)
 
-3. Select **Create Stack** to display **1. Configure Application** in the stack creation dialog.
+3. Click **Create Stack** to open the **Create Application** window. The **1. Configure Application** section will be active.
 
     ![Configure stacks in UCP](/ee/ucp/images/ucp-config-stack.png)
 
 4. Enter a name for the stack in the **Name** field.
-5. Select either **Swarm Services** or **Kubernetes Workloads** for the orchestrator mode. If you select Kubernetes, also select a namespace in the **Namespace** drop-down list.
+5. Click to indicate the Orchestrator Mode, either **Swarm Services** or **Kubernetes Workloads**. Note that if you select Kubernetes Workloads, the namespace drop-down list will display, from which you must select one of the namespaces offered. 
 
     ![Specify namespace for a stack in UCP](/ee/ucp/images/ucp-stack-namespace.png)
 
-6. Select either **Compose File** or **App Package** for the **Application File Mode**.
-7. Select **Next**.
-8. If you selected Compose file, enter or upload your `docker-compose.yml` in **2. Add Application File**.
+6. Click to indicate the Application File Mode, either **Compose File** or **App Package**.
+7. Click **Next** to open the **2. Add Application File** section.
```

Comment:
Put **Application File Mode** in bold text.

Reply:
I prefer to leave the sentence as written ("to open vs. to activate"). I think "to open" is the better choice when describing actions relating to a GUI.